src/HIE/Bios/Cradle.hs

```diff
@@ -796,6 +815,23 @@ cabalGhcDirs l cabalProject workDir = do
   where
     projectFileArgs = projectFileProcessArgs cabalProject
 
+cabalPathCompilerPath :: LogAction IO (WithSeverity Log) -> FilePath -> CradleProjectConfig -> CradleLoadResultT IO (Maybe FilePath)
```

Comment:
A little bit more documentation would be appreciated!

Reply:
Renamed so it's less ambiguous.
Really not sure what else to say about it besides "calls cabal path to get compiler's path".